formicactl/formicactl.go

```diff
@@ -0,0 +1,63 @@
+// Package formicactl implements a command line client for formica. Cobra CLI
+// is used as framework.
+package main
+
+import (
+	"net/url"
+
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/formica/controller"
+	"github.com/giantswarm/formica/fleet"
+)
+
+var (
+	globalFlags struct {
+		FleetEndpoint string
+	}
+
+	newController controller.Controller
+	newFleet      fleet.Fleet
+
+	mainCmd = &cobra.Command{
+		Use:   "formicactl",
+		Short: "orchestrate groups of unit files on Fleet clusters",
+		Long:  "orchestrate groups of unit files on Fleet clusters",
+		Run:   mainRun,
```

Comment:
Do you need to specify a run for the root command? I don't think you do - I think cobra implicitly will call show help usage if no run is specified - you'll be able to remove `mainRun` too.